src/controllers/ZakenController.ts

```diff
@@ -0,0 +1,43 @@
+import { RequestHandler, Request, Response } from "express";
+
+import got from "got";
+import { CreateHeaders } from "../utils/CreateHeader";
+
+interface Query {
+  bsn?: string;
+}
+
+export const ZakenController: RequestHandler = async (
+  req: Request,
+  res: Response
+) => {
+  const openZaakUrl = <string>process.env.OPEN_ZAAK_URL;
+  const query = req.query as Query;
+
+  const token = req.get("Authorization");
+  if (token === undefined || token === null) {
```

Comment:
`null` is the same as `undefined`. Also no value type can be implicitly coerced to `null` type, so `===` is unnecessarily.
```suggestion
  if (token == null) {
```

Reply:
I don't think they're the same, token can be either a string or undefined. A null check will not cover the undefined case.
But the null case might not be necessary.